### ISSUE TITLE
bitcode bundle could not be generated?

### ISSUE THREAD

**Issue by parcool** [blazorin/EssentialLibs#1]
```shell
ld: warning: directory not found for option '-L/Users/parcool/EssentialLibs/boost/stage/ios/lib/Release-iphoneos'
ld: warning: ignoring file /Users/parcool/CLionProjects/zlib-fork/zlib/cmake-build-iossimulatorrelease/Release-iphonesimulator/libz.a, building for iOS-arm64 but attempting to link with file built for iOS Simulator-x86_64
ld: warning: object file (/Users/parcool/EssentialLibs/boost/stage/ios/lib/libboost_filesystem.a(directory.o)) was built for newer iOS version (15.5) than being linked (11.0)
ld: bitcode bundle could not be generated because '/Users/parcool/EssentialLibs/boost/stage/ios/lib/libboost_filesystem.a(directory.o)' was built without full bitcode. All object files and libraries for bitcode must be generated from Xcode Archive or Install build for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

** BUILD FAILED **
```

I'm using this [https://github.com/leetal/ios-cmake](https://github.com/leetal/ios-cmake) ios-cmake. what is the problem?

**Comment by blazorin** [comment]
Hello!

`ld: warning: directory not found for option '-L/Users/parcool/EssentialLibs/boost/stage/ios/lib/Release-iphoneos'`

Note the correct path for iOS is [boost/stage/ios/lib](https://github.com/blazorin/EssentialLibs/tree/master/boost/stage/ios/lib) make sure CMAKE does not add any suffixes like a release mode directory. You can also try with the Cmakelists setup on the readme I provided.


`ld: bitcode bundle could not be generated because '/Users/parcool/EssentialLibs/boost/stage/ios/lib/libboost_filesystem.a(directory.o)' was built without full bitcode. All object files and libraries for bitcode must be generated from Xcode Archive or Install build for architecture arm64`

I have dealt with this last week on iOS architecture on another project, it's something that on macOS doesn't happen,
but I know what causes it, boost compilation is very big but I will fix this ASAP. 

It's strange as I am running it on my own iOS apps and did not encounter this. 
Looks like something that happens to depend on the iOS version, I use 14.7 but will update to make sure this works correctly on the last iOS version.

In other words, I will recompile the libraries with the bitcode thing fixed.
I will update here when you can download the updated libraries.

**Comment by blazorin** [comment]
Can you also check if it is just happening with the boost_filesystem? Just try to remove the -lboost_filesystem to check.

Also, try to build your app through Xcode, the output may be different. (in case you don't have Xcode project, cmake allows to create it:
`cmake <dir> -G Xcode`

I will update my phone to check if it's something related to the iOS version as I mentioned.


Let me know about that 2 things. Thanks.
